### DISCUSSION TITLE
Zustand with react-native-debugger?

### DISCUSSION THREAD

**Discussion by gkankava** [pmndrs/zustand#645] [Q&A]
how to use zustand's redux devtools with react native debugger

**Comment by karltaylor** [comment]
Has anyone actually tried this? Is it possible? Assuming it connects the same way?

**Reply by dbritto-dev** [reply]
@gkankava you can use redux dev tools config to configure it since it uses the same stack. Here you go

- https://docs.expo.dev/debugging/tools/#debugging-redux
- https://github.com/zalmoxisus/redux-devtools-extension

**Comment by hgulersen** [comment]
Dropping this here in case anyone wants to connect to `react-native-debugger` to debug their Zustand stores. Works for Expo apps as well. 

1. Install and setup the `devtools` Zustand middleware 
2. Install the following package as a dev dependency and import it as documented in the README: https://www.npmjs.com/package/react-native-devsettings?activeTab=readme
3. Open the react native debugger app
4. Open the developer window on Expo, scroll to the very bottom and click `Open React Native dev menu`
5. Click `* Debug JS Remotely` 

You should see your Zustand store.
